docs/Manifest-v1.11.toml

```diff
@@ -361,9 +361,9 @@ version = "0.10.18"
 
 [[deps.ClimaTimeSteppers]]
 deps = ["ClimaComms", "Colors", "DataStructures", "DiffEqBase", "KernelAbstractions", "Krylov", "LinearAlgebra", "LinearOperators", "NVTX", "SciMLBase", "StaticArrays"]
-git-tree-sha1 = "ff967e27a56a938d3e719bb8d2e025b086c3f808"
+git-tree-sha1 = "f03e9f4316d380cdf851ec2c4c55efbfdb064439"
 uuid = "595c0a79-7f3d-439a-bc5a-b232dc3bde79"
-version = "0.7.39"
+version = "0.8.1"
```

Comment:
Is this update needed for stable moist baroclinic wave simulation with `ARS343` time stepping scheme?
https://buildkite.com/clima/climaatmos-dot-jl-cpu-scaling/builds/45#0194ae84-64de-424f-a321-0725d5d44419

Reply:
Yes, either v0.8 or v0.7.40 (added in #3548) is required for longrun stability.